### PR TITLE
block structure for trophies page

### LINKED ISSUE #72: Migrate client services pages
This applies to the following pages:

* https://www.asprey.com/gb/en/client-services/trophies/contact-us.html
* https://www.asprey.com/gb/en/client-services/trophies/heritage-trophies.html
* https://www.asprey.com/gb/en/client-services/trophies/craftsmanship.html
* https://www.asprey.com/gb/en/client-services/trophies/trophies-awards.html
* https://www.asprey.com/gb/en/client-services/trophies/about.html
* https://www.asprey.com/gb/en/client-services/leather-care.html
* https://www.asprey.com/gb/en/client-services/leather-skins.html
* https://www.asprey.com/gb/en/customer-care/find-a-boutique.html
* https://www.asprey.com/gb/en/customer-care/find-a-boutique-usa.html
* https://www.asprey.com/gb/en/customer-care/find-a-boutique-switzerland.html
* https://www.asprey.com/gb/en/customer-care/find-a-boutique-japan.html

- [ ] Documents are already imported to Sharepoint, but feel free to simplify the content structure
- [ ] Ensure the pages are properly linked
- [ ] Please update the importer to reflect your content changes. Make sure it's reviewed by the whole team so it does not impact other content pages in an inconsistent way.

### PR DESCRIPTION
**Pull request by hannessolo**
Fix #72 

## Test URLs:
- Before: https://main--asprey--hlxsites.hlx.page/gb/en/client-services/trophies/heritage-trophies
- After:
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/client-services/trophies/heritage-trophies
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/client-services/trophies/about
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/client-services/trophies/trophies-awards
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/client-services/trophies/craftsmanship
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/customer-care/find-a-boutique
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/customer-care/find-a-boutique-switzerland
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/customer-care/find-a-boutique-usa
    - https://issue-72--asprey--hlxsites.hlx.live/gb/en/customer-care/find-a-boutique-japan

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -31,3 +31,9 @@
     order: unset;
   }
 }
+
+@media (min-width: 1200px) {
```
Comment (jckautzmann) [inline blocks/columns/columns.css:35]: isn't the Asprey breakpoint set to 1024px?

File: styles/styles.css [hunk 2]
```diff
@@ -324,3 +323,47 @@ main .section.highlight {
...
+  background: var(--clr-purple);
+  color: white;
+}
+
+main .gray-background {
+  background: var(--clr-background-highlight);
+  color: black;
+}
+
+@media (min-width: 1200px) {
```
Comment (jckautzmann) [inline styles/styles.css:341]: isn't the Asprey breakpoint set to 1024px?

File: blocks/columns/columns.css (hunk 1 above)
Comment (hannessolo) [reply blocks/columns/columns.css:35]: I didn't check the asprey breakpoints - this one is based on the fact that the section width is max 1200px https://github.com/hlxsites/asprey/blob/dfc0c61da4977ee5589b941eecdd5b1c8a35b974/styles/styles.css#L309

**Comment by hannessolo** [comment]
Following discussion with @Kronnox , we decided to introduce a new section metadata field called `template`. The template will automatically apply several css classes to a section.

The template is useful to make customized sections without overwhelming the author with a large list of css classes.

Blocks inside templated sections may change their styling depending on the section template. For example, the section template here removes padding. So the columns block inside the section template needs to add padding again.

All blocks should be written such that they still render fine outside of a templated section - but not all blocks need to render correctly when inside an arbitrary templated section.

Happy to collect feedback on this approach.

File: styles/styles.css [hunk 3]
```diff
@@ -140,16 +140,23 @@ h4, h5, h6 {
...
-h2 { font-size: var(--font-size-heading-xl) }
+h1 {
+  font-size: var(--font-size-heading-xxl);
+  font-weight: 600;
+
+}
+
+h2 {
+  font-size: var(--font-size-heading-xl);
+  font-weight: 600;
```
Comment (Kronnox) [inline styles/styles.css:158]: This won't work. For `h1` I think it's fine to do this, as it's only used in titles. `h2` however is used as a real heading that is `font-weight: 400` e.g. https://www.asprey.com/gb/en/customer-care/delivery-return.html

Comment (Kronnox) [reply styles/styles.css:158]: My current solution is the following:
```CSS
--font-size-heading-xxxl: 3.0rem;

.title h1 {
    font-size: var(--font-size-heading-xxxl);
    font-weight: 600;
    letter-spacing: 0.032rem;
}
```

**Comment by hannessolo** [comment]
Ready for review, minus some general styling changes that will be added in #87

File: styles/styles.css (hunk 3 above)
Comment (hannessolo) [reply styles/styles.css:158]: This should be fixed by making the headings bold after #87 is merged, correct?

File: blocks/hero-video/hero-video.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+export default function decorate(block) {
+  const videoTag = block.querySelector('video');
```
Comment (jckautzmann) [inline blocks/hero-video/hero-video.js:2]: nit: I'd call it `video`

File: blocks/subnav/subnav.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export default function decorate(block) {
+  block.querySelectorAll('a').forEach((a) => {
+    a.classList.add('button');
+    a.classList.add('secondary');
```
Comment (jckautzmann) [inline blocks/subnav/subnav.js:4]: nit: you could simplify with:
`a.classList.add('button', 'secondary');`

File: scripts/scripts.js [hunk 6]
```diff
@@ -10,7 +10,7 @@ import {
   waitForLCP,
   loadBlocks,
   loadCSS,
-  getMetadata,
+  getMetadata, toClassName,
```
Comment (jckautzmann) [inline scripts/scripts.js:13]: nit: add a new line

File: scripts/scripts.js [hunk 7]
```diff
@@ -62,6 +126,8 @@ async function loadFonts() {
 function buildAutoBlocks(main) {
   try {
     // No auto-blocking
```
Comment (jckautzmann) [inline scripts/scripts.js:128]: I'd remove this line

**Comment by jckautzmann** [comment]
Looks good! I just added a few nitpicking remarks :) 
For the `subnav` block authoring: I would find it easier for authors to write the links as a bulleted list, instead of separated by a whitespace.

**Comment by jckautzmann** [comment]
Page https://issue-72--asprey--hlxsites.hlx.live/gb/en/customer-care/find-a-boutique
```
Please contact us on the number below or click here if you would like to book an appointment at Asprey’s London Flagship store.
```

I noticed that the `click here` link has disappeared.

**Comment by hannessolo** [comment]
> I noticed that the click here link has disappeared.

Yes, many links were broken. It's an issue with the importer and I guess we will have a more "general" fix once I finish writing the import script for these pages.

The importer was ignoring the host, so writing links like `https:///path/` instead of `https://host/path`.

**Review by Kronnox — APPROVED** [review]

**Pull request merged**